scripts/scripts.js

```diff
@@ -47,11 +46,27 @@ export function setLanguage(pathname) {
 function buildHeroBlock(main) {
   const h1 = main.querySelector('h1');
   const picture = main.querySelector('picture');
+  const pictureParent = picture ? picture.parentNode : false;
```

Comment:
Can all this be done in the hero block itself?

Reply:
This was the initial setup for Franklin. We could move to hero.js

Reply:
all the build* and decorate* methods are in scripts.js